src/SwedbankPay/Api/Client/Version.php

```diff
@@ -48,14 +50,42 @@ protected function getVersionFromEnvironment()
             return getenv("VERSION");
         }
 
-        $path = __DIR__ . '/../../../../composer.json';
-        $contents = file_get_contents($path);
-        $data = json_decode($contents, true);
+        $composer = $this->readJson(__DIR__ . '/../../../../composer.json');
 
-        if (isset($data['version'])) {
-            return $data['version'];
+        if (isset($composer['version'])) {
+            return $composer['version'];
+        }
+
+        $composerLock = $this->readJson(__DIR__ . '/../../../../composer.lock');
+
+        if (isset($composerLock['packages'])) {
+            $packages = $composerLock['packages'];
+            foreach ($packages as $package) {
+                if (!isset($package['name']) ||
+                    $package['name'] != "swedbank-pay/swedbank-pay-sdk-php") {
+                    continue;
+                }
+
+                if (isset($package['version'])) {
+                    return $package['version'];
+                }
+            }
         }
 
         throw new ClientException('VERSION not found in environment variable, composer.json or anywhere else.');
     }
+
+
+    /**
+     * Reads the file at $path and returns it as a JSON decoded object.
+     *
+     * @param $path The path of the file to read and JSON decode.
+     * @return object The JSON decoded object.
+     */
+    private function readJson($path)
+    {
+        $path =  realpath($path);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function realpath() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)